draft-nygren-httpbis-httpssvc.md

```diff
@@ -574,6 +517,41 @@ Authoritative DNS servers SHOULD return A, AAAA, and HTTPSSVC records
 any in-bailiwick SvcDomainNames.
 
 
+# Performance optimizations
+
+For optimal performance (i.e. minimum connection setup time), clients
+SHOULD issue address (AAAA and/or A) and HTTPSSVC queries
+simultaneously, and SHOULD implement a client-side DNS cache.
+With these optimizations in place, and conforming DNS servers,
+using HTTPSSVC does not add network latency to connection setup.
+
+Using HTTPSSVC with HTTP/3 saves a roundtrip during connection setup,
+compared to HTTP over TLS 1.3 over TCP.  Performance-sensitive
+server operators SHOULD offer HTTP/3 in their HTTPSSVC records.
+
+A nonconforming recursive resolver might return an HTTPSSVC response with
+a nonempty SvcDomainName, without the corresponding address queries.  If
+all the HTTPSSVC RRs in the response have nonempty SvcDomainName values,
+and the client does not have address records for any of these values in
+its DNS cache, the client SHOULD perform an additional address query for
+the SvcDomainName.
+
```

Comment:
Do we want to mention my hack here?  (That clients SHOULD look at the set of aliases / canonical names in the A/AAAA response chain to see if they have an A/AAAA resolution of the SvcDomainName in this recent response?)  And that operators may choose to have the SvcDomainName match a name that the client would see a resolution for as part of the A/AAAA lookup?

Reply:
I think that's worth mentioning somewhere.  The client behavior might just be "decent DNS cache" behavior.

Reply:
Done